Source/DafnyLanguageServer/Program.cs

```diff
@@ -60,4 +62,15 @@ public class Program {
         .AddSerilog(Log.Logger);
     }
   }
+  internal class LogWriter : TextWriter {
```

Comment:
Since `LogWriter` is only used by the `Program` type, I suggest making it an inner `private` class.